substrate/primitives/trie/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "sp-trie"
-version = "29.0.0"
+version = "29.0.1"
```

```suggestion
version = "29.0.0"
```

Will be done release